db/migrate/20201109205227_add_locale_to_user.rb

```diff
@@ -1,5 +1,5 @@
 class AddLocaleToUser < ActiveRecord::Migration[6.0]
   def change
-    add_column :users, :locale, :string, default: 'en'
+    add_column :users, :locale, :string, default: 'ru'
```

Comment:
They will not accept this.

Maybe the following will help you:
```
class ApplicationController < ActionController::Base
  before_action :set_locale

  private

  def set_locale
    I18n.locale = :ru
  end
end
```

Reply:
@indicozy

Reply:
I know, I messed up my github tree and merged my tinkers for my application. I will revert changes after I will finish translating